src/Symfony/Component/Form/Extension/Core/CoreExtension.php

```diff
@@ -36,12 +36,21 @@ class CoreExtension extends AbstractExtension
      */
     private $choiceListFactory;
 
+    /**
+     * CoreExtension constructor.
+     *
+     * @param PropertyAccessorInterface|null  $propertyAccessor
+     * @param ChoiceListFactoryInterface|null $choiceListFactory
+     */
     public function __construct(PropertyAccessorInterface $propertyAccessor = null, ChoiceListFactoryInterface $choiceListFactory = null)
```

Comment:
We don't add phpdoc blocks when they don't add any new information that cannot be found in the signature. That's the case here, so this should be removed.